Gordon360/Services/HousingService.cs

```diff
@@ -622,23 +624,53 @@ public ApartmentApplicationViewModel GetApartmentApplication(int applicationID)
                     if (student != null)
                     {
                         // If the student information is found, create a new ApplicationViewModel and fill in its properties
-                        ApartmentApplicantViewModel applicantModel = new ApartmentApplicantViewModel();
-                        applicantModel.ApplicationID = applicationID;
-
-                        applicantModel.Profile = student;
-
-                        applicantModel.StudentID = null; // Intentionally null in this case. Do not share the ID numbers of arbitrary students with the frontend
-                        applicantModel.Username = student.AD_Username;
+                        ApartmentApplicantViewModel applicantModel = new ApartmentApplicantViewModel
+                        {
+                            ApplicationID = applicationID,
+                            Profile = student,
+                            StudentID = null, // Intentionally null in this case. Do not share the ID numbers of arbitrary students with the frontend
+                            Username = student.AD_Username,
+                            Age = null, // Not yet implemented
+                            Class = student.Class,
+                            OffCampusProgram = applicantDBModel.AprtProgram,
+                            Probation = false // Not yet implemented. This is where we will put the code to check if a student has a probation
+                        };
```

Comment:
This syntax change was actually performed automatically by the auto-formatting tool in Visual Studio, and it is quite impressive how it detected what I was trying to do here and changed it to the more concise format, including bringing the comments along.

This appears to be a syntax to selectively initialize certain properties of a new object instance upon creation, without needing to actually implement a class constructor inside the view model. This also has the added benefit of allowing you to easily provide values for only the class variables you want, leaving any other class variables either empty/null or at their default values. 

We (and other backend developers) should keep this in mind in the future since it could probably allow us to simplify the initialization of view models and other objects in other parts of the 360 project as well.

Reply:
Wow, that's really handy!